### PR TITLE
Share

### PR DESCRIPTION
**Pull request by BingyiXie**
## Checklist before requesting a review
- [x ] I have performed a self-review of my code
- [x ] My code is well documented/readable (comments when needed)
- [x ] All of my unit tests pass
- [x ] New thorough unit tests were added (if needed)
- [x ] No linting errors (run pylint) 
- [ x] Python code is formatted using Black

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -139,13 +149,71 @@ def event_detail(event_id):
     return flask.jsonify(event_data)
 
 
+@app.route("/review", methods=["GET", "POST"])
+def review():
+    "review for event"
+    if flask.request.method == "GET":
+        event_id = "vvG1zZpmTbud8h"
```
Comment (facundof13) [inline app.py]: Why is this hard coded? I think it should come from the react fetch call

File: app.py [hunk 2]
```diff
@@ -139,13 +149,71 @@ def event_detail(event_id):
...
+
+    if flask.request.method == "GET":
+        event_id = "vvG1zZpmTbud8h"
+    else:
+        event_id = flask.request.get_json()["event_id"]
+
+    event_data = get_event_detail(event_id)
+
+    msg = Message(
+        "Hello from the other side!", sender=email, recipients="bxie2@student.gsu.edu"
```
Comment (facundof13) [inline app.py]: The recipients should also come from the request. I think recipients is who receives the email, so it should be the email that comes from the front-end

File: app.py [hunk 3]
```diff
@@ -139,13 +149,71 @@ def event_detail(event_id):
...
+        "Hello, This is the event information"
+        + event_data
+        + "do you want to come with me ?"
+    )
+    mail.send(msg)
+
+    return "Message sent!"
+
+
+if __name__ == "__main__":
```
Comment (facundof13) [inline app.py]: this already exists in our code, no need to add it again. if you want to run the app in debug mode you need to add DEBUG=True to your .env file

**Review by facundof13 — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (BingyiXie) [reply app.py]: i update all the code from react fetch call

File: app.py (hunk 3 above)
Comment (BingyiXie) [reply app.py]: i deleted it

**Review by facundof13 — APPROVED** [review]

**Pull request merged**